sdk/storage/AzureStorageBlob/Source/Data/Transfer.swift

```diff
@@ -44,7 +44,8 @@ public struct TransferProgress {
     public var asFloat: Float {
         // Returning NaN sometimes results in progress appearing as 100% when it shouldn't.
         // This ensures progress will report 0%.
-        assert(bytes <= totalBytes, "Transferred bytes unexpectedly > than total bytes.")
+        // TODO: Debug concurrency instabilitites triggering this
+        // assert(bytes <= totalBytes, "Transferred bytes unexpectedly > than total bytes.")
```

Comment:
During sunny day scenarios (both with and without restart) the final bytes will equal the total expected bytes. However, core data instabilities can result in this assert being triggered, which would result in the app continuously crashing.

Reply:
That's pretty bizarre, but that's par for the course for me for CoreData :/